### ISSUE TITLE
Add foaf:homepage for Catalog

### ISSUE THREAD

**Issue by peterdesmet** [oSoc15/ipt-dcat#48]
As described in the [mapping file](https://github.com/oSoc15/ipt-dcat/blob/master/mapping.md), add `foaf:homepage` to catalog and populate with [IPT#HomepageURL](https://github.com/gbif/ipt/blob/master/src/main/java/org/gbif/ipt/model/AgentBase.java#L49).

**Comment by Amenai** [comment]
```
 /**
   * For a single organization, update its metadata. Only updates the metadata for an organisation 
   * coming from the registry, not the metadata set by the IPT administrator like can host data,
   * DOI configuration, etc.
   * @param organisation Organisation
   */ 
```

So if the organisation is not register to the GBIF he cannot get an homepage in the code (for testing mode)

**Comment by peterdesmet** [comment]
Sorry, I don't understand. You cannot retrieve the IPT HomepageURL if it (the IPT) is not registered with GBIF?

Note that you can register everything in testing mode (with a test registry).

**Comment by Amenai** [comment]
The homepage url is never set in Testing mode . The hosting organisation need to be registry to GBIF to get a homepage. Every time we want to get the homepage of the ipt it's return "null"

**Comment by peterdesmet** [comment]
OK, so on `null` the homepage is not included in DCAT and on something other than `null` it is included in DCAT (but this cannot be tested)? If so, leave this issue open with the label "to test".

**Comment by simon-vc** [comment]
Here is my thought
The DCAT specification defines the homepage of the catalogn which would mean the baseURL of the application
I am going to use this one

Secondly, when a homepage is defined for the hosting organization, I will include it as a homapage of the publisher

**Comment by peterdesmet** [comment]
Great!

**Comment by peterdesmet** [comment]
Catalog homepage as IPT URL confirmed fixed in `v1.2`.

The organization homepage currently shows a dummy value:

```
<http://www.gbif.org/publisher/0a16da09-7719-40de-8d4f-56a15ed52fb6#Organization> a foaf:Agent ; foaf:name "Test Organisation #1" ; foaf:homepage <[http://homepage.com]> .
```

**Comment by peterdesmet** [comment]
Will report organization homepage in separate issue.
